package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "capacitor-branch-deep-links",
-  "version": "7.1.0",
+  "version": "7.2.0",
```

Comment:
Since this is a major dependency change I would suggest also bumping the major of our sdk to reflect

Reply:
Just updated it to 8.0.0